src/applications/hca/tests/e2e/hca-shortform.cypress.spec.js

```diff
@@ -1,11 +1,3 @@
-/**
- * [TestRail-integrated] Spec for Medical Copays
- * @testrailinfo projectId 10
- * @testrailinfo suiteId 11
- * @testrailinfo groupId 3267
- * @testrailinfo runName HCA-e2e-AIQ
- */
-
```

Comment:
Was this test rail metadata meant to be removed?

Reply:
@Mottie yes, I added by mistake when I originally created the file.